src/app/action-playground/action-playground.component.ts

```diff
@@ -105,6 +115,46 @@ export class ActionPlaygroundComponent implements OnInit {
     }
 
     sortList(key) {
-        this.list = this.sortingService.keySortByAlphabetical(this.list, key);
+        this.list = this._sortingService.keySortByAlphabetical(this.list, key);
     }
+
+    // TODO: Will be replaced by login process from action module
+    /*
+    login() {
```

Comment:
So here we will use the login / logout component instead?

Reply:
Yes exactly. It doesn't exist yet in the ui-lib action module. The login/logout component will use the session service